src/api/cli/create_rawtx.go

```diff
@@ -22,8 +22,8 @@ type UnspentOut struct {
 }
 
 type SendAmount struct {
-	Addr  string `json:"addr"`  // send to address
-	Coins uint64 `json:"coins"` // send amount
+	Addr  string  `json:"addr"`  // send to address
+	Coins float64 `json:"coins"` // send amount
```

Comment:
do not use floats

Reply:
everything should be droplets
if necessary to use floating point style, must use string and convert to uint64
Can use this package for string to decimals
https://github.com/shopspring/decimal
but never convert to floats